docs/dashboards/custom-component.md

```diff
@@ -382,3 +382,69 @@ Dashboards.board('container', {
     }]
 });
 ```
+
+
+## Custom Component with data from the DataConnector
+
+The example below shows how to develop a custom component that fetches data from the DataConnector, processes and displays it on the dashboard.
+
+The custom component is created by extending the `HTMLComponent` class and it displays the total revenue.
+
+<iframe style="width: 100%; height: 700px; border: none;" src='https://www.highcharts.com/samples/embed/dashboards/components/custom-component-data-connector' allow="fullscreen"></iframe>
+
+The DataConnector is registered on the `load` that is why we need to execute and await the `super.load()` method first to make sure that the DataConnector is registered. An important part is that the `load` method is `async` because we need to wait for the data to be fetched and processed.
+
+When the data is ready, the `getTotalRevenue` method is used to calculate the total revenue. The `getElementsFromString` method is used to parse the HTML string into the AST-like object. The `render` method is used to render the component on the dashboard.
+
+```js
+const { ComponentRegistry } = Dashboards,
+    HTMLComponent = ComponentRegistry.types.HTML;
+
+class TotalRevenueHTML extends HTMLComponent {
+    constructor(cell, options) {
+        super(cell, options);
+
+        this.type = 'TotalRevenueHTML';
+
+        return this;
+    }
+
+    async load() {
+        await super.load();
+        const revenue = this.getTotalRevenue();
+
+        this.elements = this.getElementsFromString(
+            `
+            <div class="revenue">
+                <p class="title">Total revenue</p>
+                <p class="value">${revenue} €</p>
+            </div>
+            `
+        );
+        this.render();
+    }
+
+    getTotalRevenue() {
+        const connector = this.connectorHandlers[0].connector;
```

Comment:
Everything is fine here, but just FYI I added a method to `dash/hc-mutliple-connectors` which you can also use here: `this.getFirstConnector()`. If you think it would look better, up to you.